CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - The new Kuma addon installs the Kuma service mesh.
 - `--kong-ingress-controller-image` selects the ingress controller image
   for the Kong addon.
+- The metallb tag is now `v0.12.1`.
```

Comment:
`v0.15.0` has already been released. This change should land in a new release: either `v0.16.0` or `v0.15.1` where I believe the latter seems more appropriate for this.

```suggestion
- Update metallb to `v0.12.1`.
```